srt.go

```diff
@@ -40,9 +40,14 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 		line = strings.TrimSpace(scanner.Text())
 		lineNum++
 
+		if lineNum == 1 {
```

Comment:
Can you add a comment `// Remove BOM header` above `if lineNum == 1 {`  ?

Reply:
added in https://github.com/asticode/go-astisub/pull/22/commits/489f53adc2376337082e86637e00f39a74a929b2